src/components/SnmpCheck/SnmpCheck.tsx

```diff
@@ -1,75 +1,115 @@
-import { Button, LoadingOverlay, NumberInput, Stack, TextInput } from "@mantine/core";
+import { Button, NumberInput, Stack, TextInput } from "@mantine/core";
 import { useForm } from "@mantine/form";
-import { useCallback, useState } from "react";
+import { useCallback, useEffect, useState } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
-import { UserGuide } from "../UserGuide/UserGuide";
-import { SaveOutputToTextFile } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { RenderComponent } from "../UserGuide/UserGuide";
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+import InstallationModal from "../InstallationModal/InstallationModal";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 
-const title = "SnmpCheck";
-const description_userguide =
-    "The SNMP Check tool enables you to perform SNMP (Simple Network Management Protocol) checks on a specific IP " +
-    "address or hostname and port. SNMP is a widely used protocol for managing and monitoring network devices." +
-    " \n\nTo perform a scan, follow these steps: \n\n" +
-    "Step 1: Enter the IP address or hostname of the target device\n\n" +
-    "Step 2 (Optional): Specify a target port number (default port: 161). \n\n" +
-    "Step 3: Click the 'Scan' button to initiate the SNMP check.\n\n" +
-    "The tool will establish a connection to the specified device and retrieve SNMP-related information, such as system details, interfaces, and performance metrics. The results will be displayed in the console below.";
-("Please note that SNMP checks require appropriate permissions and credentials. Ensure that you have the necessary access rights before performing a scan.");
-
-interface FormValues {
+/**
+ * FormValues defines the structure of the object used to hold form state in the SNMP check component.
+ * @field ip: The ip address or hostname
+ * @field port: The SNMP port, defaults to 161 if not specified.
+ */
+interface FormValuesType {
     ip: string;
     port: number;
 }
+// Component Constants.
+const title = "SnmpCheck"; // Title of the tool.
+const description = // Description of the tool.
+    "The SNMP Check tool enables you to perform SNMP (Simple Network Management Protocol) checks on a specific IP " +
```

Comment:
Be consistent with the naming of the tool. Is it SnmpCheck or SNMP Check?

Reply:
Hi Modified to make it consistent.